charts/ctlog/templates/ctlog-backend-config.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if eq .Values.server.ingress.className "gce" }}
```

Comment:
i think we should add here as well the check if the ingress is enabled as well `.Values.server.ingress.enabled`